assignments/run_callback/run_callback.js

```diff
@@ -1,5 +1,9 @@
 function RunCallback(a, b, cb) {
-    // Write you code here, you need to add a and b, pass the result into callback function cb as argument return the result
+    let res = a + b
+    return cb(res)
+}
+function cb(a){
```

Comment:
As your test cases are passing perfectly but this function is not at used anywhere without this function also the code will work perfectly fine.